apps/site/src/app/(home)/sections/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,31 @@
+import { getQuestions } from "./getQuestions";
+import FAQAccordion from "./FAQAccordion";
+
+import { PortableText } from "@portabletext/react";
+import styles from "./FAQ.module.scss";
+
+const FAQ = async () => {
+	const questions = await getQuestions();
+	const faq = questions[0]["faqs"].map(({ _key, question, answer }) => ({
+		_key: _key,
```

Comment:
Nitpick: you can just state `_key` here instead of `_key: _key` for the sake of brevity.